# 緊急停止モードから待機モードに移行(SCIBOT-Xのみ)
---

